aspnetcore/signalr/configuration.md

```diff
@@ -278,18 +278,46 @@ Additional options for configuring timeout and keep-alive behavior are available
 
 | Option | Default value | Description |
 | ------ | ------------- | ----------- |
-| `ServerTimeout` | 30 seconds (30,000 milliseconds) | Timeout for server activity. If the server hasn't sent a message in this interval, the client considers the server disconnected and triggers the `Closed` event (`onclose` in JavaScript). This value must be large enough for a ping message to be sent from the server **and** received by the client within the timeout interval. The recommended value is a number at least double the server's `KeepAliveInterval` value to allow time for pings to arrive. |
```

Comment:
7 lines above this:
> Additional options for configuring timeout and keep-alive behavior are available on the `HubConnection` object itself:

The methods are on `HubConnectionBuilder`, so this would need to be reworked if you go this approach.

Reply:
Resolved in latest commit.  Thanks very much Brennan for the great help.